lfucg-exactions/client/js/reducers/tokenReducer.js

```diff
@@ -10,9 +10,21 @@ const tokenReducer = (state = {}, action) => {
     } = action;
     switch (endpoint) {
     case LOGIN:
+        try {
+            localStorage.setItem('Token', action.response.key);
+        } catch (e) {
+            document.cookie = `Token=${action.response.key}`;
+        }
+        global.Authorization = action.response.key;
+        return action.response;
     case ME:
         return action.response;
     case LOGOUT:
+        try {
+            localStorage.removeItem('Token');
+        } catch (e) {
+            document.cookie = 'Token=""';
```

Comment:
Cookies don't get deleted this way. You need to expire them. Research clearing cookies with Javascript